include/Output.hpp

```diff
@@ -19,8 +19,13 @@ class Output : public OutputListeners
   ~Output() = default;
 
   void setFrameListener();
-  void setFullscreen(bool fullscreen);
-  bool getFullscreen() const;
+  void setFullscreenView(View *view) noexcept;
+
+  View *getFullscreenView() const noexcept
```

Comment:
Why is this getter here and the setter in the corresponding .cpp file ?

Reply:
So that gets may be inlined. Since simple gets also generally don't change, this doesn't cost compile time and not having this in the .cpp does not hinder understanding.